tests/lib/testers/rule-tester.js

```diff
@@ -126,7 +123,16 @@ describe("RuleTester", () => {
                     { code: "var foo = bar;", errors: [{ message: "Bad error message." }] }
                 ]
             });
-        }, /Bad var\.((.*==)|(.*strictEqual)).*Bad error message/);
+        }, err => {
```

Comment:
Instead of using a function here, could we just provide the string as the second argument?

```js
assert.throws(() => {
    // ...
}, new nodeAssert.AssertionError({ /* ... */ }).message);
```

(This would work if we use `chai.assert.throws` rather than Node's `assert.throws`.)

Reply:
That does not work. The reason is that the operator has to be detected from the thrown error, since that changed over time as well.